artifactory/services/buildinfo.go

```diff
@@ -87,3 +95,37 @@ func (bis *BuildInfoService) PublishBuildInfo(build *buildinfo.BuildInfo, projec
 	log.Debug("Artifactory response:", resp.Status)
 	return summary, nil
 }
+
+func (bis *BuildInfoService) DeleteBuildInfo(build *buildinfo.BuildInfo, projectKey string, buildNumberFrequency int) error {
+	params := CreateDeleteBuildInfoBody(build, projectKey, buildNumberFrequency)
+	content, err := json.Marshal(params)
+	if err != nil {
+		return err
```

Comment:
```suggestion
		return errorutils.CheckError(err)
```
We wrap error where they originally returned by this function. It serves a debugging functionality, which we can go over in our next session.

Reply:
we have seen this but never understood its purpose properly